content/source/docs/enterprise/private/capacity.html.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Capacity and Performance"
+sidebar_current: "docs-enterprise2-private-installer-capacity"
+---
+
+# Private Terraform Enterprise Capacity and Performance (Installer)
+
+The maximum capacity and performance of Private Terraform Enterprise (PTFE) is dependent entirely on the resources
+provided by the Linux instance it is installed on. There are a few settings that allow the capacity that PTFE
+uses to be adjusted to suit the instance.
+
+## Memory + Concurrency
+
+The amount of memory to allocate to a Terraform run and the number of concurrent runs are the primary elements in
+understanding capacity above the base services.
+
+By default, PTFE allocates 256 MB of memory to each Terraform run, with a default concurrency of 10 parallel runs.
+Therefore, by default PTFE requires 2.6 GB of memory reserved for runs.
```

Comment:
What happens if not enough memory is available? Does PTFE reserve less, raise a warning, or not work at all?

Reply:
It doesn't provide any feedback about that kind of misconfiguration atm.